### PR TITLE
Changes to make tests happy

### PR DESCRIPTION
**Pull request by danderson00**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@danderson00__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (dalander). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: sdk/managed/test/Microsoft.WindowsAzure.MobileServices.Test/FunctionalTests/FromSdkTests/MobileServiceTableTest.cs [hunk 1]
```diff
@@ -578,124 +578,124 @@ public class MobileServiceTableGenericFunctionalTests : FunctionalTestBase
             await stringIdTable.DeleteAsync(item);
         }
 
-        [AsyncTestMethod]
-        [Tag("NodeRuntimeOnly")]
-        public async Task AsyncTableOperationsWithStringIdAgainstIntegerIdTable()
```
Comment (danderson00) [inline sdk/managed/test/Microsoft.WindowsAzure.MobileServices.Test/FunctionalTests/FromSdkTests/MobileServiceTableTest.cs]: This test is node only, tests really specific error messages and is largely irrelevant - too much effort to fix. We'll consider testing this scenario when we refactor.

**Comment by phvannor** [comment]
Sorry but this will have to go against net-client now.

**Comment by danderson00** [comment]
You are not being clear. Do you mean this test should be run against the .NET backend? If we intend to retain this test, it requires refactoring.

**Comment by danderson00** [comment]
It's OK, the penny dropped. I'm assuming now that you mean the tests have been moved to net-client. The duplication should be removed ASAP!

**Comment by phvannor** [comment]
once the release is done, I didn't want to break any last minute needs

**Pull request closed without merging**